{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "if(window['d3'] === undefined ||\n",
       "   window['Nyaplot'] === undefined){\n",
       "    var path = {\"d3\":\"https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min\",\"downloadable\":\"http://cdn.rawgit.com/domitry/d3-downloadable/master/d3-downloadable\"};\n",
       "\n",
       "\n",
       "\n",
       "    var shim = {\"d3\":{\"exports\":\"d3\"},\"downloadable\":{\"exports\":\"downloadable\"}};\n",
       "\n",
       "    require.config({paths: path, shim:shim});\n",
       "\n",
       "\n",
       "require(['d3'], function(d3){window['d3']=d3;console.log('finished loading d3');require(['downloadable'], function(downloadable){window['downloadable']=downloadable;console.log('finished loading downloadable');\n",
       "\n",
       "\tvar script = d3.select(\"head\")\n",
       "\t    .append(\"script\")\n",
       "\t    .attr(\"src\", \"http://cdn.rawgit.com/domitry/Nyaplotjs/master/release/nyaplot.js\")\n",
       "\t    .attr(\"async\", true);\n",
       "\n",
       "\tscript[0][0].onload = script[0][0].onreadystatechange = function(){\n",
       "\n",
       "\n",
       "\t    var event = document.createEvent(\"HTMLEvents\");\n",
       "\t    event.initEvent(\"load_nyaplot\",false,false);\n",
       "\t    window.dispatchEvent(event);\n",
       "\t    console.log('Finished loading Nyaplotjs');\n",
       "\n",
       "\t};\n",
       "\n",
       "\n",
       "});});\n",
       "}\n"
      ],
      "text/plain": [
       "\"if(window['d3'] === undefined ||\\n   window['Nyaplot'] === undefined){\\n    var path = {\\\"d3\\\":\\\"https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min\\\",\\\"downloadable\\\":\\\"http://cdn.rawgit.com/domitry/d3-downloadable/master/d3-downloadable\\\"};\\n\\n\\n\\n    var shim = {\\\"d3\\\":{\\\"exports\\\":\\\"d3\\\"},\\\"downloadable\\\":{\\\"exports\\\":\\\"downloadable\\\"}};\\n\\n    require.config({paths: path, shim:shim});\\n\\n\\nrequire(['d3'], function(d3){window['d3']=d3;console.log('finished loading d3');require(['downloadable'], function(downloadable){window['downloadable']=downloadable;console.log('finished loading downloadable');\\n\\n\\tvar script = d3.select(\\\"head\\\")\\n\\t    .append(\\\"script\\\")\\n\\t    .attr(\\\"src\\\", \\\"http://cdn.rawgit.com/domitry/Nyaplotjs/master/release/nyaplot.js\\\")\\n\\t    .attr(\\\"async\\\", true);\\n\\n\\tscript[0][0].onload = script[0][0].onreadystatechange = function(){\\n\\n\\n\\t    var event = document.createEvent(\\\"HTMLEvents\\\");\\n\\t    event.initEvent(\\\"load_nyaplot\\\",false,false);\\n\\t    window.dispatchEvent(event);\\n\\t    console.log('Finished loading Nyaplotjs');\\n\\n\\t};\\n\\n\\n});});\\n}\\n\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "true"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "require 'cbioportal'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "#<CBioPortal:0x0000555e1abb8e60>"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "C"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<b> Daru::DataFrame(239x12) </b>\n",
       "<table>\n",
       "  <thead>\n",
       "  \n",
       "    <tr>\n",
       "      <th></th>\n",
       "      \n",
       "        <th>allSampleCount</th>\n",
       "      \n",
       "        <th>cancerTypeId</th>\n",
       "      \n",
       "        <th>citation</th>\n",
       "      \n",
       "        <th>description</th>\n",
       "      \n",
       "        <th>groups</th>\n",
       "      \n",
       "        <th>importDate</th>\n",
       "      \n",
       "        <th>name</th>\n",
       "      \n",
       "        <th>pmid</th>\n",
       "      \n",
       "        <th>publicStudy</th>\n",
       "      \n",
       "        <th>shortName</th>\n",
       "      \n",
       "        <th>status</th>\n",
       "      \n",
       "        <th>studyId</th>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "</thead>\n",
       "  <tbody>\n",
       "  \n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      \n",
       "        <td>34</td>\n",
       "      \n",
       "        <td>blca</td>\n",
       "      \n",
       "        <td>Al-Ahmadie et al. Nat Genet 2016</td>\n",
       "      \n",
       "        <td>Genomic profiling of plasmacytoid-variant (signet ring cell) tumors of the bladder, published set</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>Bladder Cancer, Plasmacytoid Variant (MSKCC, Nat Genet 2016)</td>\n",
       "      \n",
       "        <td>26901067</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Bladder PV (MSKCC)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>blca_plasmacytoid_mskcc_2016</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      \n",
       "        <td>40</td>\n",
       "      \n",
       "        <td>hnsc</td>\n",
       "      \n",
       "        <td>Pickering et al. Cancer Discov 2013</td>\n",
       "      \n",
       "        <td>Comprehensive profiling of 40 oral squamous cell carcinoma samples from MD Anderson.</td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>Oral Squamous Cell Carcinoma (MD Anderson, Cancer Discov 2013)</td>\n",
       "      \n",
       "        <td>23619168</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Head & neck (MDA)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>hnsc_mdanderson_2013</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      \n",
       "        <td>46</td>\n",
       "      \n",
       "        <td>liad</td>\n",
       "      \n",
       "        <td>Pilati et al. Cancer Cell 2014</td>\n",
       "      \n",
       "        <td>Mutation data from whole-exome sequencing (WXS) of 46 liver tumor and corresponding nontumor DNA.</td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>Hepatocellular Adenoma (Inserm, Cancer Cell 2014)</td>\n",
       "      \n",
       "        <td>24735922</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Liad (Inserm 2014)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>liad_inserm_fr_2014</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      \n",
       "        <td>56</td>\n",
       "      \n",
       "        <td>nbl</td>\n",
       "      \n",
       "        <td>Peifer et al. Nature 2015</td>\n",
       "      \n",
       "        <td>Mutation data from whole-genome sequencing of 56 neuroblastoma samples</td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>Neuroblastoma (Broad, Nat Genet 2013)</td>\n",
       "      \n",
       "        <td>26466568</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>NBL (Cologne 2015)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>nbl_ucologne_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      \n",
       "        <td>10</td>\n",
       "      \n",
       "        <td>panet</td>\n",
       "      \n",
       "        <td>Cao et al. Nat Commun 2013</td>\n",
       "      \n",
       "        <td>Whole exome sequencing of 10 benign insulinoma tumors. <A HREF=\"http://www.nature.com/ncomms/2013/131210/ncomms3810/extref/ncomms3810-s1.pdf\"></td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>Insulinoma (Shanghai, Nat Commun 2013)</td>\n",
       "      \n",
       "        <td>24326773</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Panet (Shanghai 2013)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>panet_shanghai_2013</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>5</td>\n",
       "      \n",
       "        <td>29</td>\n",
       "      \n",
       "        <td>mcl</td>\n",
       "      \n",
       "        <td>Bea et al. PNAS 2013</td>\n",
       "      \n",
       "        <td>Whole exome sequencing of 29 primary mantle cell lymphomas.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>Mantle Cell Lymphoma (IDIBIPS, PNAS 2013)</td>\n",
       "      \n",
       "        <td>24145436</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>MCL (IDIBIPS 2013)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>mcl_idibips_2013</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>6</td>\n",
       "      \n",
       "        <td>29</td>\n",
       "      \n",
       "        <td>mds</td>\n",
       "      \n",
       "        <td>Yoshida et al. Nature 2011</td>\n",
       "      \n",
       "        <td>Whole exome sequencing of 29 myelodysplasia samples from the University of Tokyo.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>Myelodysplasia (Tokyo, Nature 2011)</td>\n",
       "      \n",
       "        <td>21909114</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>MDS (Tokyo)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>mds_tokyo_2011</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>7</td>\n",
       "      \n",
       "        <td>22</td>\n",
       "      \n",
       "        <td>plmeso</td>\n",
       "      \n",
       "        <td>Guo et al. Cancer Res 2015</td>\n",
       "      \n",
       "        <td>Mutation data from whole-exome sequencing of 22 malignant pleural mesotheliomas (PLMESO) and matched blood samples.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>Malignant Pleural Mesothelioma (NYU, Cancer Res 2015)</td>\n",
       "      \n",
       "        <td>25488749</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>PLMESO (NYU 2015)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>plmeso_nyu_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>8</td>\n",
       "      \n",
       "        <td>99</td>\n",
       "      \n",
       "        <td>paad</td>\n",
       "      \n",
       "        <td>Biankin et al. Nature 2012</td>\n",
       "      \n",
       "        <td>Comprehensive profiling of 99 pancreatic cancer samples. Generated by ICGC.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>Pancreatic Adenocarcinoma (ICGC, Nature 2012)</td>\n",
       "      \n",
       "        <td>23103869</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Pancreas (ICGC)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>paad_icgc</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>9</td>\n",
       "      \n",
       "        <td>477</td>\n",
       "      \n",
       "        <td>prad</td>\n",
       "      \n",
       "        <td>Fraser et al. Nature 2017</td>\n",
       "      \n",
       "        <td>Comprehensive genomic profiling of 477 Prostate Adenocarcinoma samples from CPC-GENE and public data sets, including TCGA-PRAD.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>Genomic Hallmarks of Prostate Adenocarcinoma (CPC-GENE, Nature 2017)</td>\n",
       "      \n",
       "        <td>28068672</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>PRAD Hallmarks (CPCG-GENE, 2017)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>prad_cpcg_2017</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>10</td>\n",
       "      \n",
       "        <td>176</td>\n",
       "      \n",
       "        <td>prad</td>\n",
       "      \n",
       "        <td>Kumar et al. Nat Med 2016</td>\n",
       "      \n",
       "        <td>Comprehensive profiling of prostate cancer samples. Generated by Peter Nelson's lab at the Fred Hutchinson Cancer Research Center.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-05-15 04:00:00</td>\n",
       "      \n",
       "        <td>Prostate Adenocarcinoma (Fred Hutchinson CRC, Nat Med 2016)</td>\n",
       "      \n",
       "        <td>26928463</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Prostate (FHCRC, 2016)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>prad_fhcrc</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>11</td>\n",
       "      \n",
       "        <td>32</td>\n",
       "      \n",
       "        <td>pact</td>\n",
       "      \n",
       "        <td>Wu et al. PNAS 2011</td>\n",
       "      \n",
       "        <td>Whole-exome sequencing of major neoplastic cyst types of the pancreas: 8 serous cystadenomas (SCAs), 8 intraductal papillary mucinous neoplasms (IPMNs), 8 mucinous cystic neoplasms (MCNs), and 8 solid pseudopapillary neoplasms (SPNs).</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-05-15 04:00:00</td>\n",
       "      \n",
       "        <td>Cystic Tumor of the Pancreas (Johns Hopkins, PNAS 2011</td>\n",
       "      \n",
       "        <td>22158988</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Pancreas (JHU 2011)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>pact_jhu_2011</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>12</td>\n",
       "      \n",
       "        <td>96</td>\n",
       "      \n",
       "        <td>past</td>\n",
       "      \n",
       "        <td>Jones et al. Nat Genet 2013</td>\n",
       "      \n",
       "        <td>Whole-genome sequencing of 96 pilocytic astrocytomasatic alterations in pilocytic astrocytoma conducted by the International Cancer Genome Consortium (ICGC) PedBrain Tumor Project.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-05-15 04:00:00</td>\n",
       "      \n",
       "        <td>Whole-genome sequencing of pilocytic astrocytomasatic (Nat Genetics, 2013)</td>\n",
       "      \n",
       "        <td>23817572</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>PAST (Nat Gen, 2013)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>past_dkfz_heidelberg_2013</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>13</td>\n",
       "      \n",
       "        <td>41</td>\n",
       "      \n",
       "        <td>nsclc</td>\n",
       "      \n",
       "        <td>Vavalˆ et al. Lung Cancer 2017</td>\n",
       "      \n",
       "        <td>The retrospective study using NGS approach for genetic characterization of NSCLC young adult patients</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-05-15 04:00:00</td>\n",
       "      \n",
       "        <td>Genetic Characterization of NSCLC young adult patients ( University of Turin, Lung Cancer 2016)</td>\n",
       "      \n",
       "        <td>27346245</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>UNITO (Lung Cancer, 2016)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>nsclc_unito_2016</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>14</td>\n",
       "      \n",
       "        <td>243</td>\n",
       "      \n",
       "        <td>hcc</td>\n",
       "      \n",
       "        <td>Schulze et al. Nat Genet 2015</td>\n",
       "      \n",
       "        <td>Mutation data from whole-exome sequencing (WXS) of 243 liver tumors identified mutational signatures associated with specific risk factors.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-05-17 04:00:00</td>\n",
       "      \n",
       "        <td>Hepatocellular Carcinomas (Inserm, Nat Genet 2015)</td>\n",
       "      \n",
       "        <td>25822088</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>HCC (Inserm, 2015)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>hcc_inserm_fr_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>15</td>\n",
       "      \n",
       "        <td>98</td>\n",
       "      \n",
       "        <td>panet</td>\n",
       "      \n",
       "        <td>Scarpa et al. Nature 2017</td>\n",
       "      \n",
       "        <td>Whole-genome sequencing of 102 primary PanNETs and the defined genomic events that characterize their pathogenesis.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-05-17 04:00:00</td>\n",
       "      \n",
       "        <td>Whole-Genome Sequencing of Pancreatic Neuroendocrine Tumors (Nature, 2017)</td>\n",
       "      \n",
       "        <td>28199314</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>ACRNET (Nature, 2017)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>panet_arcnet_2017</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>16</td>\n",
       "      \n",
       "        <td>66</td>\n",
       "      \n",
       "        <td>skcm</td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>Next generation sequencing (NGS) of pre-treatment tumor to identify co-altered genes that predict for intrinsic resistance to BRAF inhibitor therapy in melanoma patients as a prelude to rational combination strategies.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-10-02 04:00:00</td>\n",
       "      \n",
       "        <td>Next generation sequencing (NGS) of pre-treatment metastatic melanoma samples (MSK, JCO Precision Oncology 2017)</td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>SKCM (Vanderbilt/Mskcc)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>skcm_vanderbilt_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>17</td>\n",
       "      \n",
       "        <td>22</td>\n",
       "      \n",
       "        <td>odg</td>\n",
       "      \n",
       "        <td>Thomas et al. Neuro Oncol 2017</td>\n",
       "      \n",
       "        <td>Genomic characterization using next generation sequencing (NGS) in Anaplastic Oligodendroglioma and Anaplastic Oligoastrocytomas.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-10-02 04:00:00</td>\n",
       "      \n",
       "        <td>NGS in Anaplastic Oligodendroglioma and Anaplastic Oligoastrocytomas tumors (MSK, Neuro Oncol 2017)</td>\n",
       "      \n",
       "        <td>28472509</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Glioma MSK (Neuro Oncol 2017)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>odg_msk_2017</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>18</td>\n",
       "      \n",
       "        <td>39</td>\n",
       "      \n",
       "        <td>skcm</td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>Genetic Characterization of 38 pretreatment (pembrolizumab and nivolumab) melanoma tumors.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-10-02 04:00:00</td>\n",
       "      \n",
       "        <td>Whole-exome sequences (WES) of pretreatment melanoma tumors (UCLA, Cell 2016)</td>\n",
       "      \n",
       "        <td>26997480</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>SKCM UCLA (Cell 2016)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>skcm_ucla_2016</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>19</td>\n",
       "      \n",
       "        <td>38</td>\n",
       "      \n",
       "        <td>mel</td>\n",
       "      \n",
       "        <td>Liang et al. Genome Res 2017</td>\n",
       "      \n",
       "        <td>Comprehensive genomic and transcriptomic analysis of ALM patients.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-10-02 04:00:00</td>\n",
       "      \n",
       "        <td>Paired-exome sequencing of acral melanoma (TGEN, Genome Res 2017)</td>\n",
       "      \n",
       "        <td>28373299</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>TSAM</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>mel_tsam_liang_2017</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>20</td>\n",
       "      \n",
       "        <td>915</td>\n",
       "      \n",
       "        <td>luad</td>\n",
       "      \n",
       "        <td>Jordan EJ et al. Cancer Discov 2017</td>\n",
       "      \n",
       "        <td>Prospective molecular characterization of lung adenocarcinomas for efficient patient matching.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-10-02 04:00:00</td>\n",
       "      \n",
       "        <td>MSK-IMPACT Clinical Sequencing Cohort for Non-Small Cell Cancer (MSK, Cancer Discovery 2017)</td>\n",
       "      \n",
       "        <td>28336552</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>LUAD (MSK,2017)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>lung_msk_2017</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>21</td>\n",
       "      \n",
       "        <td>504</td>\n",
       "      \n",
       "        <td>prad</td>\n",
       "      \n",
       "        <td>Abida et al, JCO Precision Oncology 2017</td>\n",
       "      \n",
       "        <td>Targeted sequencing of clinical cases via MSK-IMPACT for prostate cancer (Abida et al, JCO Precision Oncology 2017).</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-10-04 04:00:00</td>\n",
       "      \n",
       "        <td>MSK-IMPACT Clinical Sequencing Cohort in Prostate Cancer (MSK, JCO Precision Oncology 2017)</td>\n",
       "      \n",
       "        <td></td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>MSK-IMPACT Prostate</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>prad_mskcc_2017</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>22</td>\n",
       "      \n",
       "        <td>62</td>\n",
       "      \n",
       "        <td>urcc</td>\n",
       "      \n",
       "        <td>Chen et al. Nat Commun 2016</td>\n",
       "      \n",
       "        <td>Molecular Analysis of Unclassified Renal Cell Carcinoma</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-10-09 04:00:00</td>\n",
       "      \n",
       "        <td>Targeted gene sequencing in 62 high-grade primary Unclassified Renal Cell Carcinoma (MSK, Nature 2016)</td>\n",
       "      \n",
       "        <td>27713405</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Renal unclass (MSKCC)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>urcc_mskcc_2016</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>23</td>\n",
       "      \n",
       "        <td>97</td>\n",
       "      \n",
       "        <td>blca</td>\n",
       "      \n",
       "        <td>Iyer et al. JCO 2013</td>\n",
       "      \n",
       "        <td>Comprehensive profiling of 97 bladder cancer samples, including targeted sequencing of 15 cancer genes. Iyer et al., JCO in press.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-11-04 04:00:00</td>\n",
       "      \n",
       "        <td>Bladder Cancer (MSKCC, JCO 2013)</td>\n",
       "      \n",
       "        <td>23897969</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Bladder (MSKCC 2012)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>blca_mskcc_solit_2012</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>24</td>\n",
       "      \n",
       "        <td>109</td>\n",
       "      \n",
       "        <td>blca</td>\n",
       "      \n",
       "        <td>Kim et al. Eur Urol 2015</td>\n",
       "      \n",
       "        <td>Genomic Predictors of Survival in Patients with High-grade Urothelial Carcinoma of the Bladder. Targeted sequencing of 109 tumor samples (MSKCC, European Urology 2014).</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-11-04 04:00:00</td>\n",
       "      \n",
       "        <td>Bladder Cancer (MSKCC, Eur Urol 2014)</td>\n",
       "      \n",
       "        <td>25092538</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Bladder (MSKCC 2014)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>blca_mskcc_solit_2014</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>25</td>\n",
       "      \n",
       "        <td>65</td>\n",
       "      \n",
       "        <td>brca</td>\n",
       "      \n",
       "        <td>Shah et al. Nature</td>\n",
       "      \n",
       "        <td>Comprehensive profiling of 65 breast cancer samples. Generated by the British Columbia Cancer Research Centre.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-11-04 04:00:00</td>\n",
       "      \n",
       "        <td>Breast Invasive Carcinoma (British Columbia, Nature 2012)</td>\n",
       "      \n",
       "        <td>22495314</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Breast (BCCRC 2012)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>brca_bccrc</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>26</td>\n",
       "      \n",
       "        <td>99</td>\n",
       "      \n",
       "        <td>blca</td>\n",
       "      \n",
       "        <td>Guo et al. Nat Genet 2012</td>\n",
       "      \n",
       "        <td>Exome and whole-genome sequencing of 99 bladder cancer samples. Generated by BGI.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-11-06 05:00:00</td>\n",
       "      \n",
       "        <td>Bladder Urothelial Carcinoma (BGI, Nat Genet 2013)</td>\n",
       "      \n",
       "        <td>24121792</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Bladder (BGI 2013)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>blca_bgi</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>27</td>\n",
       "      \n",
       "        <td>100</td>\n",
       "      \n",
       "        <td>brca</td>\n",
       "      \n",
       "        <td>Stephens et al. Nature 2012</td>\n",
       "      \n",
       "        <td>Comprehensive profiling of 100 breast cancer samples. Generated by the Sanger Institute.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-11-10 05:00:00</td>\n",
       "      \n",
       "        <td>Breast Invasive Carcinoma (Sanger, Nature 2012)</td>\n",
       "      \n",
       "        <td>22722201</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Breast (Sanger)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>brca_sanger</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>28</td>\n",
       "      \n",
       "        <td>825</td>\n",
       "      \n",
       "        <td>brca</td>\n",
       "      \n",
       "        <td>TCGA, Nature 2012</td>\n",
       "      \n",
       "        <td><a href=\"http://cancergenome.nih.gov/\">The Cancer Genome Atlas (TCGA)</a> Breast Invasive Carcinoma project. 825 cases.<br><i>Nature 2012.</i> <a href=\"https://tcga-data.nci.nih.gov/docs/publications/brca_2012/\">Raw data via the TCGA Data Portal</a>.</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-11-10 05:00:00</td>\n",
       "      \n",
       "        <td>Breast Invasive Carcinoma (TCGA, Nature 2012)</td>\n",
       "      \n",
       "        <td>23000897</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>Breast (TCGA pub)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>brca_tcga_pub</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>29</td>\n",
       "      \n",
       "        <td>79</td>\n",
       "      \n",
       "        <td>ccrcc</td>\n",
       "      \n",
       "        <td>Gerlinger et al. Nat Genet 2014</td>\n",
       "      \n",
       "        <td>Mutation data from multi region exome sequencing of 10 clear cell renal cell carcinomas (ccRCCs)</td>\n",
       "      \n",
       "        <td>PUBLIC</td>\n",
       "      \n",
       "        <td>2017-11-10 05:00:00</td>\n",
       "      \n",
       "        <td>Multiregion Sequencing of Clear Cell Renal Cell Carcinoma (IRC, Nat Genet 2014)</td>\n",
       "      \n",
       "        <td>24487277</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>ccRCC (IRC)</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>ccrcc_irc_2014</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "\n",
       "  \n",
       "    <tr>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "    </tr>\n",
       "\n",
       "    \n",
       "\n",
       "    <tr>\n",
       "      <td>238</td>\n",
       "        \n",
       "          <td>91</td>\n",
       "        \n",
       "          <td>difg</td>\n",
       "        \n",
       "          <td></td>\n",
       "        \n",
       "          <td>Targeted Sequencing of 91 cfDNA samples (86 samples with matched normals) from 42 glioma patients.</td>\n",
       "        \n",
       "          <td>PUBLIC</td>\n",
       "        \n",
       "          <td>2019-01-18 05:00:00</td>\n",
       "        \n",
       "          <td>Glioma (MSK, 2018)</td>\n",
       "        \n",
       "          <td></td>\n",
       "        \n",
       "          <td>true</td>\n",
       "        \n",
       "          <td>DIFG(MSK 2018)</td>\n",
       "        \n",
       "          <td>0</td>\n",
       "        \n",
       "          <td>glioma_msk_2018</td>\n",
       "        \n",
       "    </tr>\n",
       "  \n",
       "</tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "#<Daru::DataFrame(239x12)>\n",
       "            allSampleC cancerType   citation descriptio     groups importDate       name       pmid publicStud  shortName     status    studyId\n",
       "          0         34       blca Al-Ahmadie Genomic pr     PUBLIC        nil Bladder Ca   26901067       true Bladder PV          0 blca_plasm\n",
       "          1         40       hnsc Pickering  Comprehens                   nil Oral Squam   23619168       true Head & nec          0 hnsc_mdand\n",
       "          2         46       liad Pilati et  Mutation d                   nil Hepatocell   24735922       true Liad (Inse          0 liad_inser\n",
       "          3         56        nbl Peifer et  Mutation d                   nil Neuroblast   26466568       true NBL (Colog          0 nbl_ucolog\n",
       "          4         10      panet Cao et al. Whole exom                   nil Insulinoma   24326773       true Panet (Sha          0 panet_shan\n",
       "          5         29        mcl Bea et al. Whole exom     PUBLIC        nil Mantle Cel   24145436       true MCL (IDIBI          0 mcl_idibip\n",
       "          6         29        mds Yoshida et Whole exom     PUBLIC        nil Myelodyspl   21909114       true MDS (Tokyo          0 mds_tokyo_\n",
       "          7         22     plmeso Guo et al. Mutation d     PUBLIC        nil Malignant    25488749       true PLMESO (NY          0 plmeso_nyu\n",
       "          8         99       paad Biankin et Comprehens     PUBLIC        nil Pancreatic   23103869       true Pancreas (          0  paad_icgc\n",
       "          9        477       prad Fraser et  Comprehens     PUBLIC        nil Genomic Ha   28068672       true PRAD Hallm          0 prad_cpcg_\n",
       "         10        176       prad Kumar et a Comprehens     PUBLIC 2017-05-15 Prostate A   26928463       true Prostate (          0 prad_fhcrc\n",
       "         11         32       pact Wu et al.  Whole-exom     PUBLIC 2017-05-15 Cystic Tum   22158988       true Pancreas (          0 pact_jhu_2\n",
       "         12         96       past Jones et a Whole-geno     PUBLIC 2017-05-15 Whole-geno   23817572       true PAST (Nat           0 past_dkfz_\n",
       "         13         41      nsclc Vavalˆ et  The retros     PUBLIC 2017-05-15 Genetic Ch   27346245       true UNITO (Lun          0 nsclc_unit\n",
       "         14        243        hcc Schulze et Mutation d     PUBLIC 2017-05-17 Hepatocell   25822088       true HCC (Inser          0 hcc_inserm\n",
       "        ...        ...        ...        ...        ...        ...        ...        ...        ...        ...        ...        ...        ..."
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "C.get_all_studies"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<b> Daru::DataFrame(853x6) </b>\n",
       "<table>\n",
       "  <thead>\n",
       "  \n",
       "    <tr>\n",
       "      <th></th>\n",
       "      \n",
       "        <th>name</th>\n",
       "      \n",
       "        <th>clinicalTrialKeywords</th>\n",
       "      \n",
       "        <th>dedicatedColor</th>\n",
       "      \n",
       "        <th>shortName</th>\n",
       "      \n",
       "        <th>parent</th>\n",
       "      \n",
       "        <th>cancerTypeId</th>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "</thead>\n",
       "  <tbody>\n",
       "  \n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      \n",
       "        <td>Aggressive Angiomyxoma</td>\n",
       "      \n",
       "        <td>aggressive angiomyxoma</td>\n",
       "      \n",
       "        <td>LightYellow</td>\n",
       "      \n",
       "        <td>AA</td>\n",
       "      \n",
       "        <td>soft_tissue</td>\n",
       "      \n",
       "        <td>aa</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      \n",
       "        <td>Anaplastic Astrocytoma</td>\n",
       "      \n",
       "        <td>anaplastic astrocytoma</td>\n",
       "      \n",
       "        <td>Gray</td>\n",
       "      \n",
       "        <td>AASTR</td>\n",
       "      \n",
       "        <td>difg</td>\n",
       "      \n",
       "        <td>aastr</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      \n",
       "        <td>Activated B-cell Type</td>\n",
       "      \n",
       "        <td>activated b-cell type</td>\n",
       "      \n",
       "        <td>LimeGreen</td>\n",
       "      \n",
       "        <td>ABC</td>\n",
       "      \n",
       "        <td>dlbclnos</td>\n",
       "      \n",
       "        <td>abc</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      \n",
       "        <td>Acute Basophilic Leukemia</td>\n",
       "      \n",
       "        <td>acute basophilic leukemia</td>\n",
       "      \n",
       "        <td>LightSalmon</td>\n",
       "      \n",
       "        <td>ABL</td>\n",
       "      \n",
       "        <td>amlnos</td>\n",
       "      \n",
       "        <td>abl</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      \n",
       "        <td>Adrenocortical Adenoma</td>\n",
       "      \n",
       "        <td>adrenocortical adenoma</td>\n",
       "      \n",
       "        <td>Purple</td>\n",
       "      \n",
       "        <td>ACA</td>\n",
       "      \n",
       "        <td>adrenal_gland</td>\n",
       "      \n",
       "        <td>aca</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>5</td>\n",
       "      \n",
       "        <td>Adenoid Cystic Breast Cancer</td>\n",
       "      \n",
       "        <td>adenoid cystic breast cancer</td>\n",
       "      \n",
       "        <td>HotPink</td>\n",
       "      \n",
       "        <td>ACBC</td>\n",
       "      \n",
       "        <td>brca</td>\n",
       "      \n",
       "        <td>acbc</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>6</td>\n",
       "      \n",
       "        <td>Adrenocortical Carcinoma</td>\n",
       "      \n",
       "        <td>adrenocortical carcinoma</td>\n",
       "      \n",
       "        <td>Purple</td>\n",
       "      \n",
       "        <td>ACC</td>\n",
       "      \n",
       "        <td>adrenal_gland</td>\n",
       "      \n",
       "        <td>acc</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>7</td>\n",
       "      \n",
       "        <td>Acinic Cell Carcinoma</td>\n",
       "      \n",
       "        <td>acinic cell carcinoma</td>\n",
       "      \n",
       "        <td>DarkRed</td>\n",
       "      \n",
       "        <td>ACCC</td>\n",
       "      \n",
       "        <td>saca</td>\n",
       "      \n",
       "        <td>accc</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>8</td>\n",
       "      \n",
       "        <td>Atypical Chronic Myeloid Leukemia, BCR-ABL1-</td>\n",
       "      \n",
       "        <td>atypical chronic myeloid leukemia, bcr-abl1-</td>\n",
       "      \n",
       "        <td>LightSalmon</td>\n",
       "      \n",
       "        <td>ACML</td>\n",
       "      \n",
       "        <td>mds/mpn</td>\n",
       "      \n",
       "        <td>acml</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>9</td>\n",
       "      \n",
       "        <td>Acinar Cell Carcinoma, NOS</td>\n",
       "      \n",
       "        <td>acinar cell carcinoma, nos</td>\n",
       "      \n",
       "        <td>Black</td>\n",
       "      \n",
       "        <td>ACN</td>\n",
       "      \n",
       "        <td>cup</td>\n",
       "      \n",
       "        <td>acn</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>10</td>\n",
       "      \n",
       "        <td>Craniopharyngioma, Adamantinomatous Type</td>\n",
       "      \n",
       "        <td>craniopharyngioma, adamantinomatous type</td>\n",
       "      \n",
       "        <td>Gray</td>\n",
       "      \n",
       "        <td>ACPG</td>\n",
       "      \n",
       "        <td>selt</td>\n",
       "      \n",
       "        <td>acpg</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>11</td>\n",
       "      \n",
       "        <td>Atypical Choroid Plexus Papilloma</td>\n",
       "      \n",
       "        <td>atypical choroid plexus papilloma</td>\n",
       "      \n",
       "        <td>Gray</td>\n",
       "      \n",
       "        <td>ACPP</td>\n",
       "      \n",
       "        <td>cpt</td>\n",
       "      \n",
       "        <td>acpp</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>12</td>\n",
       "      \n",
       "        <td>Acral Melanoma</td>\n",
       "      \n",
       "        <td>acral melanoma</td>\n",
       "      \n",
       "        <td>Black</td>\n",
       "      \n",
       "        <td>ACRM</td>\n",
       "      \n",
       "        <td>mel</td>\n",
       "      \n",
       "        <td>acrm</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>13</td>\n",
       "      \n",
       "        <td>Adenoid Cystic Carcinoma</td>\n",
       "      \n",
       "        <td>adenoid cystic carcinoma</td>\n",
       "      \n",
       "        <td>DarkRed</td>\n",
       "      \n",
       "        <td>ACYC</td>\n",
       "      \n",
       "        <td>saca</td>\n",
       "      \n",
       "        <td>acyc</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>14</td>\n",
       "      \n",
       "        <td>Adamantinoma</td>\n",
       "      \n",
       "        <td>adamantinoma</td>\n",
       "      \n",
       "        <td>White</td>\n",
       "      \n",
       "        <td>ADMA</td>\n",
       "      \n",
       "        <td>bone</td>\n",
       "      \n",
       "        <td>adma</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>15</td>\n",
       "      \n",
       "        <td>Adenocarcinoma, NOS</td>\n",
       "      \n",
       "        <td>adenocarcinoma, nos</td>\n",
       "      \n",
       "        <td>Black</td>\n",
       "      \n",
       "        <td>ADNOS</td>\n",
       "      \n",
       "        <td>cup</td>\n",
       "      \n",
       "        <td>adnos</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>16</td>\n",
       "      \n",
       "        <td>Aggressive Digital Papillary Adenocarcinoma</td>\n",
       "      \n",
       "        <td>aggressive digital papillary adenocarcinoma</td>\n",
       "      \n",
       "        <td>Black</td>\n",
       "      \n",
       "        <td>ADPA</td>\n",
       "      \n",
       "        <td>skin</td>\n",
       "      \n",
       "        <td>adpa</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>17</td>\n",
       "      \n",
       "        <td>Adrenal Gland</td>\n",
       "      \n",
       "        <td>adrenal gland</td>\n",
       "      \n",
       "        <td>Purple</td>\n",
       "      \n",
       "        <td>ADRENAL_GLAND</td>\n",
       "      \n",
       "        <td>tissue</td>\n",
       "      \n",
       "        <td>adrenal_gland</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>18</td>\n",
       "      \n",
       "        <td>Sweat Gland Carcinoma/Apocrine Eccrine Carcinoma</td>\n",
       "      \n",
       "        <td>sweat gland carcinoma/apocrine eccrine carcinoma</td>\n",
       "      \n",
       "        <td>Black</td>\n",
       "      \n",
       "        <td>AECA</td>\n",
       "      \n",
       "        <td>skin</td>\n",
       "      \n",
       "        <td>aeca</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>19</td>\n",
       "      \n",
       "        <td>Angiomatoid Fibrous Histiocytoma</td>\n",
       "      \n",
       "        <td>angiomatoid fibrous histiocytoma</td>\n",
       "      \n",
       "        <td>LightYellow</td>\n",
       "      \n",
       "        <td>AFH</td>\n",
       "      \n",
       "        <td>soft_tissue</td>\n",
       "      \n",
       "        <td>afh</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>20</td>\n",
       "      \n",
       "        <td>Atypical Fibroxanthoma</td>\n",
       "      \n",
       "        <td>atypical fibroxanthoma</td>\n",
       "      \n",
       "        <td>Black</td>\n",
       "      \n",
       "        <td>AFX</td>\n",
       "      \n",
       "        <td>skin</td>\n",
       "      \n",
       "        <td>afx</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>21</td>\n",
       "      \n",
       "        <td>Anal Gland Adenocarcinoma</td>\n",
       "      \n",
       "        <td>anal gland adenocarcinoma</td>\n",
       "      \n",
       "        <td>SaddleBrown</td>\n",
       "      \n",
       "        <td>AGA</td>\n",
       "      \n",
       "        <td>bowel</td>\n",
       "      \n",
       "        <td>aga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>22</td>\n",
       "      \n",
       "        <td>Anaplastic Ganglioglioma</td>\n",
       "      \n",
       "        <td>anaplastic ganglioglioma</td>\n",
       "      \n",
       "        <td>Gray</td>\n",
       "      \n",
       "        <td>AGNG</td>\n",
       "      \n",
       "        <td>encg</td>\n",
       "      \n",
       "        <td>agng</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>23</td>\n",
       "      \n",
       "        <td>Alpha Heavy-Chain Disease</td>\n",
       "      \n",
       "        <td>alpha heavy-chain disease</td>\n",
       "      \n",
       "        <td>LimeGreen</td>\n",
       "      \n",
       "        <td>AHCD</td>\n",
       "      \n",
       "        <td>mbn</td>\n",
       "      \n",
       "        <td>ahcd</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>24</td>\n",
       "      \n",
       "        <td>Adenocarcinoma In Situ</td>\n",
       "      \n",
       "        <td>adenocarcinoma in situ</td>\n",
       "      \n",
       "        <td>Black</td>\n",
       "      \n",
       "        <td>AIS</td>\n",
       "      \n",
       "        <td>other</td>\n",
       "      \n",
       "        <td>ais</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>25</td>\n",
       "      \n",
       "        <td>Angioimmunoblastic T-Cell Lymphoma</td>\n",
       "      \n",
       "        <td>angioimmunoblastic t-cell lymphoma</td>\n",
       "      \n",
       "        <td>LimeGreen</td>\n",
       "      \n",
       "        <td>AITL</td>\n",
       "      \n",
       "        <td>mtnn</td>\n",
       "      \n",
       "        <td>aitl</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>26</td>\n",
       "      \n",
       "        <td>Acute Leukemias of Ambiguous Lineage</td>\n",
       "      \n",
       "        <td>acute leukemias of ambiguous lineage</td>\n",
       "      \n",
       "        <td>LightSalmon</td>\n",
       "      \n",
       "        <td>ALAL</td>\n",
       "      \n",
       "        <td>mnm</td>\n",
       "      \n",
       "        <td>alal</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>27</td>\n",
       "      \n",
       "        <td>Anaplastic Large Cell Lymphoma</td>\n",
       "      \n",
       "        <td>anaplastic large cell lymphoma</td>\n",
       "      \n",
       "        <td>LimeGreen</td>\n",
       "      \n",
       "        <td>ALCL</td>\n",
       "      \n",
       "        <td>mtnn</td>\n",
       "      \n",
       "        <td>alcl</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>28</td>\n",
       "      \n",
       "        <td>Anaplastic Large-Cell Lymphoma ALK Negative</td>\n",
       "      \n",
       "        <td>anaplastic large-cell lymphoma alk negative</td>\n",
       "      \n",
       "        <td>LimeGreen</td>\n",
       "      \n",
       "        <td>ALCLALKN</td>\n",
       "      \n",
       "        <td>alcl</td>\n",
       "      \n",
       "        <td>alclalkn</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>29</td>\n",
       "      \n",
       "        <td>Anaplastic Large-Cell Lymphoma ALK Positive</td>\n",
       "      \n",
       "        <td>anaplastic large-cell lymphoma alk positive</td>\n",
       "      \n",
       "        <td>LimeGreen</td>\n",
       "      \n",
       "        <td>ALCLALKP</td>\n",
       "      \n",
       "        <td>alcl</td>\n",
       "      \n",
       "        <td>alclalkp</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "\n",
       "  \n",
       "    <tr>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "    </tr>\n",
       "\n",
       "    \n",
       "\n",
       "    <tr>\n",
       "      <td>852</td>\n",
       "        \n",
       "          <td>Wilms' Tumor</td>\n",
       "        \n",
       "          <td>wilms' tumor</td>\n",
       "        \n",
       "          <td>Orange</td>\n",
       "        \n",
       "          <td>WT</td>\n",
       "        \n",
       "          <td>kidney</td>\n",
       "        \n",
       "          <td>wt</td>\n",
       "        \n",
       "    </tr>\n",
       "  \n",
       "</tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "#<Daru::DataFrame(853x6)>\n",
       "                  name clinicalTr dedicatedC  shortName     parent cancerType\n",
       "          0 Aggressive aggressive LightYello         AA soft_tissu         aa\n",
       "          1 Anaplastic anaplastic       Gray      AASTR       difg      aastr\n",
       "          2 Activated  activated   LimeGreen        ABC   dlbclnos        abc\n",
       "          3 Acute Baso acute baso LightSalmo        ABL     amlnos        abl\n",
       "          4 Adrenocort adrenocort     Purple        ACA adrenal_gl        aca\n",
       "          5 Adenoid Cy adenoid cy    HotPink       ACBC       brca       acbc\n",
       "          6 Adrenocort adrenocort     Purple        ACC adrenal_gl        acc\n",
       "          7 Acinic Cel acinic cel    DarkRed       ACCC       saca       accc\n",
       "          8 Atypical C atypical c LightSalmo       ACML    mds/mpn       acml\n",
       "          9 Acinar Cel acinar cel      Black        ACN        cup        acn\n",
       "         10 Craniophar craniophar       Gray       ACPG       selt       acpg\n",
       "         11 Atypical C atypical c       Gray       ACPP        cpt       acpp\n",
       "         12 Acral Mela acral mela      Black       ACRM        mel       acrm\n",
       "         13 Adenoid Cy adenoid cy    DarkRed       ACYC       saca       acyc\n",
       "         14 Adamantino adamantino      White       ADMA       bone       adma\n",
       "        ...        ...        ...        ...        ...        ...        ..."
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "C.get_all_cancer_types"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<b> Daru::DataFrame(100000x6) </b>\n",
       "<table>\n",
       "  <thead>\n",
       "  \n",
       "    <tr>\n",
       "      <th></th>\n",
       "      \n",
       "        <th>chromosome</th>\n",
       "      \n",
       "        <th>cytoband</th>\n",
       "      \n",
       "        <th>entrezGeneId</th>\n",
       "      \n",
       "        <th>hugoGeneSymbol</th>\n",
       "      \n",
       "        <th>length</th>\n",
       "      \n",
       "        <th>type</th>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "</thead>\n",
       "  <tbody>\n",
       "  \n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      \n",
       "        <td>11</td>\n",
       "      \n",
       "        <td>11q13.1</td>\n",
       "      \n",
       "        <td>-79138</td>\n",
       "      \n",
       "        <td>SPINDOC_PS121</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      \n",
       "        <td>11</td>\n",
       "      \n",
       "        <td>11q13.1</td>\n",
       "      \n",
       "        <td>-79137</td>\n",
       "      \n",
       "        <td>SPINDOC_PS55</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      \n",
       "        <td>16</td>\n",
       "      \n",
       "        <td>16p12.3</td>\n",
       "      \n",
       "        <td>-79136</td>\n",
       "      \n",
       "        <td>VPS35L_PS1002</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>1q32.1</td>\n",
       "      \n",
       "        <td>-79135</td>\n",
       "      \n",
       "        <td>INAVA_PS485</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      \n",
       "        <td>9</td>\n",
       "      \n",
       "        <td>9q31.3</td>\n",
       "      \n",
       "        <td>-79134</td>\n",
       "      \n",
       "        <td>ECPAS_PS1838</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>5</td>\n",
       "      \n",
       "        <td>17</td>\n",
       "      \n",
       "        <td>17p13.3</td>\n",
       "      \n",
       "        <td>-79133</td>\n",
       "      \n",
       "        <td>TRARG1_PS57</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>6</td>\n",
       "      \n",
       "        <td>10</td>\n",
       "      \n",
       "        <td>10p15.1</td>\n",
       "      \n",
       "        <td>-79132</td>\n",
       "      \n",
       "        <td>FBH1_PS79</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>7</td>\n",
       "      \n",
       "        <td>19</td>\n",
       "      \n",
       "        <td>19p13.3</td>\n",
       "      \n",
       "        <td>-79131</td>\n",
       "      \n",
       "        <td>YJU2_PT301</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>8</td>\n",
       "      \n",
       "        <td>9</td>\n",
       "      \n",
       "        <td>9q31.3</td>\n",
       "      \n",
       "        <td>-79130</td>\n",
       "      \n",
       "        <td>ECPAS_PT1218</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>9</td>\n",
       "      \n",
       "        <td>17</td>\n",
       "      \n",
       "        <td>17p13.3</td>\n",
       "      \n",
       "        <td>-79129</td>\n",
       "      \n",
       "        <td>TRARG1_PS66</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>10</td>\n",
       "      \n",
       "        <td>17</td>\n",
       "      \n",
       "        <td>17p13.3</td>\n",
       "      \n",
       "        <td>-79128</td>\n",
       "      \n",
       "        <td>TRARG1_PS77</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>11</td>\n",
       "      \n",
       "        <td>19</td>\n",
       "      \n",
       "        <td>19p13.3</td>\n",
       "      \n",
       "        <td>-79127</td>\n",
       "      \n",
       "        <td>YJU2_PS213_S220</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>12</td>\n",
       "      \n",
       "        <td>19</td>\n",
       "      \n",
       "        <td>19p13.3</td>\n",
       "      \n",
       "        <td>-79126</td>\n",
       "      \n",
       "        <td>YJU2_PS211_S213</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>13</td>\n",
       "      \n",
       "        <td>22</td>\n",
       "      \n",
       "        <td>22q13.2</td>\n",
       "      \n",
       "        <td>-79125</td>\n",
       "      \n",
       "        <td>PHETA2_PS179</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>14</td>\n",
       "      \n",
       "        <td>9</td>\n",
       "      \n",
       "        <td>9q31.3</td>\n",
       "      \n",
       "        <td>-79124</td>\n",
       "      \n",
       "        <td>ECPAS_PS1249</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>15</td>\n",
       "      \n",
       "        <td>10</td>\n",
       "      \n",
       "        <td>10p15.1</td>\n",
       "      \n",
       "        <td>-79123</td>\n",
       "      \n",
       "        <td>FBH1_PS52</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>16</td>\n",
       "      \n",
       "        <td>10</td>\n",
       "      \n",
       "        <td>10p15.1</td>\n",
       "      \n",
       "        <td>-79122</td>\n",
       "      \n",
       "        <td>FBH1_PS53</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>17</td>\n",
       "      \n",
       "        <td>10</td>\n",
       "      \n",
       "        <td>10p15.1</td>\n",
       "      \n",
       "        <td>-79121</td>\n",
       "      \n",
       "        <td>FBH1_PS50</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>18</td>\n",
       "      \n",
       "        <td>12</td>\n",
       "      \n",
       "        <td>12q24.12</td>\n",
       "      \n",
       "        <td>-79120</td>\n",
       "      \n",
       "        <td>PHETA1_PS192</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>19</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>1q32.1</td>\n",
       "      \n",
       "        <td>-79119</td>\n",
       "      \n",
       "        <td>INAVA_PS161</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>20</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>1q32.1</td>\n",
       "      \n",
       "        <td>-79118</td>\n",
       "      \n",
       "        <td>INAVA_PT301_T306</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>21</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>1q32.1</td>\n",
       "      \n",
       "        <td>-79117</td>\n",
       "      \n",
       "        <td>INAVA_PS139</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>22</td>\n",
       "      \n",
       "        <td>9</td>\n",
       "      \n",
       "        <td>9q31.3</td>\n",
       "      \n",
       "        <td>-79116</td>\n",
       "      \n",
       "        <td>ECPAS_PS1215</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>23</td>\n",
       "      \n",
       "        <td>17</td>\n",
       "      \n",
       "        <td>17p13.3</td>\n",
       "      \n",
       "        <td>-79115</td>\n",
       "      \n",
       "        <td>TRARG1_PT46</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>24</td>\n",
       "      \n",
       "        <td>9</td>\n",
       "      \n",
       "        <td>9q31.3</td>\n",
       "      \n",
       "        <td>-79114</td>\n",
       "      \n",
       "        <td>ECPAS_PS1125</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>25</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>1q32.1</td>\n",
       "      \n",
       "        <td>-79113</td>\n",
       "      \n",
       "        <td>INAVA_PS531</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>26</td>\n",
       "      \n",
       "        <td>10</td>\n",
       "      \n",
       "        <td>10p15.1</td>\n",
       "      \n",
       "        <td>-79112</td>\n",
       "      \n",
       "        <td>FBH1_PS50_S52</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>27</td>\n",
       "      \n",
       "        <td>10</td>\n",
       "      \n",
       "        <td>10p15.1</td>\n",
       "      \n",
       "        <td>-79111</td>\n",
       "      \n",
       "        <td>FBH1_PS50_S53</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>28</td>\n",
       "      \n",
       "        <td>22</td>\n",
       "      \n",
       "        <td>22q13.2</td>\n",
       "      \n",
       "        <td>-79110</td>\n",
       "      \n",
       "        <td>PHETA2_PS207</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>29</td>\n",
       "      \n",
       "        <td>10</td>\n",
       "      \n",
       "        <td>10p15.1</td>\n",
       "      \n",
       "        <td>-79109</td>\n",
       "      \n",
       "        <td>FBH1_PS25</td>\n",
       "      \n",
       "        <td>0</td>\n",
       "      \n",
       "        <td>phosphoprotein</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "\n",
       "  \n",
       "    <tr>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "    </tr>\n",
       "\n",
       "    \n",
       "\n",
       "    <tr>\n",
       "      <td>99999</td>\n",
       "        \n",
       "          <td>11</td>\n",
       "        \n",
       "          <td>11q12.1</td>\n",
       "        \n",
       "          <td>282769</td>\n",
       "        \n",
       "          <td>OR10AF1P</td>\n",
       "        \n",
       "          <td></td>\n",
       "        \n",
       "          <td>pseudo</td>\n",
       "        \n",
       "    </tr>\n",
       "  \n",
       "</tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "#<Daru::DataFrame(100000x6)>\n",
       "            chromosome   cytoband entrezGene hugoGeneSy     length       type\n",
       "          0         11    11q13.1     -79138 SPINDOC_PS          0 phosphopro\n",
       "          1         11    11q13.1     -79137 SPINDOC_PS          0 phosphopro\n",
       "          2         16    16p12.3     -79136 VPS35L_PS1          0 phosphopro\n",
       "          3          1     1q32.1     -79135 INAVA_PS48          0 phosphopro\n",
       "          4          9     9q31.3     -79134 ECPAS_PS18          0 phosphopro\n",
       "          5         17    17p13.3     -79133 TRARG1_PS5          0 phosphopro\n",
       "          6         10    10p15.1     -79132  FBH1_PS79          0 phosphopro\n",
       "          7         19    19p13.3     -79131 YJU2_PT301          0 phosphopro\n",
       "          8          9     9q31.3     -79130 ECPAS_PT12          0 phosphopro\n",
       "          9         17    17p13.3     -79129 TRARG1_PS6          0 phosphopro\n",
       "         10         17    17p13.3     -79128 TRARG1_PS7          0 phosphopro\n",
       "         11         19    19p13.3     -79127 YJU2_PS213          0 phosphopro\n",
       "         12         19    19p13.3     -79126 YJU2_PS211          0 phosphopro\n",
       "         13         22    22q13.2     -79125 PHETA2_PS1          0 phosphopro\n",
       "         14          9     9q31.3     -79124 ECPAS_PS12          0 phosphopro\n",
       "        ...        ...        ...        ...        ...        ...        ..."
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "C.get_all_genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<b> Daru::DataFrame(886x7) </b>\n",
       "<table>\n",
       "  <thead>\n",
       "  \n",
       "    <tr>\n",
       "      <th></th>\n",
       "      \n",
       "        <th>molecularAlterationType</th>\n",
       "      \n",
       "        <th>datatype</th>\n",
       "      \n",
       "        <th>name</th>\n",
       "      \n",
       "        <th>description</th>\n",
       "      \n",
       "        <th>showProfileInAnalysisTab</th>\n",
       "      \n",
       "        <th>molecularProfileId</th>\n",
       "      \n",
       "        <th>studyId</th>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "</thead>\n",
       "  <tbody>\n",
       "  \n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      \n",
       "        <td>COPY_NUMBER_ALTERATION</td>\n",
       "      \n",
       "        <td>DISCRETE</td>\n",
       "      \n",
       "        <td>Putative copy-number alterations from GISTIC</td>\n",
       "      \n",
       "        <td>Putative copy-number from GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015_cna</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      \n",
       "        <td>MUTATION_EXTENDED</td>\n",
       "      \n",
       "        <td>MAF</td>\n",
       "      \n",
       "        <td>Mutations</td>\n",
       "      \n",
       "        <td>Whole exome sequencing of 12 breast AdCCs.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015_mutations</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      \n",
       "        <td>PROTEIN_LEVEL</td>\n",
       "      \n",
       "        <td>LOG2-VALUE</td>\n",
       "      \n",
       "        <td>Protein expression (RPPA)</td>\n",
       "      \n",
       "        <td>Protein expression measured by reverse-phase protein array</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>acc_tcga_rppa</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      \n",
       "        <td>PROTEIN_LEVEL</td>\n",
       "      \n",
       "        <td>Z-SCORE</td>\n",
       "      \n",
       "        <td>Protein expression Z-scores (RPPA)</td>\n",
       "      \n",
       "        <td>Protein expression, measured by reverse-phase protein array, Z-scores</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acc_tcga_rppa_Zscores</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      \n",
       "        <td>COPY_NUMBER_ALTERATION</td>\n",
       "      \n",
       "        <td>DISCRETE</td>\n",
       "      \n",
       "        <td>Putative copy-number alterations from GISTIC</td>\n",
       "      \n",
       "        <td>Putative copy-number calls on 90 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acc_tcga_gistic</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>5</td>\n",
       "      \n",
       "        <td>MRNA_EXPRESSION</td>\n",
       "      \n",
       "        <td>CONTINUOUS</td>\n",
       "      \n",
       "        <td>mRNA expression (RNA Seq V2 RSEM)</td>\n",
       "      \n",
       "        <td>Expression levels for 20532 genes in 79 acc cases (RNA Seq V2 RSEM).</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>acc_tcga_rna_seq_v2_mrna</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>6</td>\n",
       "      \n",
       "        <td>MRNA_EXPRESSION</td>\n",
       "      \n",
       "        <td>Z-SCORE</td>\n",
       "      \n",
       "        <td>mRNA Expression z-Scores (RNA Seq V2 RSEM)</td>\n",
       "      \n",
       "        <td>mRNA z-Scores (RNA Seq V2 RSEM) compared to the expression distribution of each gene tumors that are diploid for this gene.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acc_tcga_rna_seq_v2_mrna_median_Zscores</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>7</td>\n",
       "      \n",
       "        <td>COPY_NUMBER_ALTERATION</td>\n",
       "      \n",
       "        <td>CONTINUOUS</td>\n",
       "      \n",
       "        <td>Relative linear copy-number values</td>\n",
       "      \n",
       "        <td>Relative linear copy-number values for each gene (from Affymetrix SNP6).</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>acc_tcga_linear_CNA</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>8</td>\n",
       "      \n",
       "        <td>METHYLATION</td>\n",
       "      \n",
       "        <td>CONTINUOUS</td>\n",
       "      \n",
       "        <td>Methylation (HM450)</td>\n",
       "      \n",
       "        <td>Methylation (HM450) beta-values for genes in 80 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression.</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>acc_tcga_methylation_hm450</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>9</td>\n",
       "      \n",
       "        <td>MUTATION_EXTENDED</td>\n",
       "      \n",
       "        <td>MAF</td>\n",
       "      \n",
       "        <td>Mutations</td>\n",
       "      \n",
       "        <td>Mutation data from whole exome sequencing. Mutation packager: ACC/20160128/gdac.broadinstitute.org_ACC.Mutation_Packager_Calls.Level_3.2016012800.0.0.tar.gz.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acc_tcga_mutations</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>10</td>\n",
       "      \n",
       "        <td>COPY_NUMBER_ALTERATION</td>\n",
       "      \n",
       "        <td>DISCRETE</td>\n",
       "      \n",
       "        <td>Putative copy-number alterations from GISTIC</td>\n",
       "      \n",
       "        <td>Putative copy-number from GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018_gistic</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>11</td>\n",
       "      \n",
       "        <td>MRNA_EXPRESSION</td>\n",
       "      \n",
       "        <td>CONTINUOUS</td>\n",
       "      \n",
       "        <td>mRNA Expression, RSEM (Batch normalized from Illumina HiSeq_RNASeqV2)</td>\n",
       "      \n",
       "        <td>mRNA Expression, RSEM (Batch normalized from Illumina HiSeq_RNASeqV2)</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018_rna_seq_v2_mrna</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>12</td>\n",
       "      \n",
       "        <td>MRNA_EXPRESSION</td>\n",
       "      \n",
       "        <td>Z-SCORE</td>\n",
       "      \n",
       "        <td>mRNA Expression Zscores, RSEM (Batch normalized from Illumina HiSeq_RNASeqV2)</td>\n",
       "      \n",
       "        <td>mRNA Expression Zscores, RSEM (Batch normalized from Illumina HiSeq_RNASeqV2)</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018_rna_seq_v2_mrna_median_Zscores</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>13</td>\n",
       "      \n",
       "        <td>COPY_NUMBER_ALTERATION</td>\n",
       "      \n",
       "        <td>LOG2-VALUE</td>\n",
       "      \n",
       "        <td>Log2 copy-number values</td>\n",
       "      \n",
       "        <td>Log2 copy-number values for each gene (from Affymetrix SNP6).</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018_log2CNA</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>14</td>\n",
       "      \n",
       "        <td>MUTATION_EXTENDED</td>\n",
       "      \n",
       "        <td>MAF</td>\n",
       "      \n",
       "        <td>Mutations</td>\n",
       "      \n",
       "        <td>Mutation data from exome sequencing data that were compiled using handpicked individual MAFs taken from the individual tumor type AWG lists.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018_mutations</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>15</td>\n",
       "      \n",
       "        <td>COPY_NUMBER_ALTERATION</td>\n",
       "      \n",
       "        <td>DISCRETE</td>\n",
       "      \n",
       "        <td>Putative copy-number alterations from GISTIC</td>\n",
       "      \n",
       "        <td>Putative copy-number from GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acyc_fmi_2014_cna</td>\n",
       "      \n",
       "        <td>acyc_fmi_2014</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>16</td>\n",
       "      \n",
       "        <td>MUTATION_EXTENDED</td>\n",
       "      \n",
       "        <td>MAF</td>\n",
       "      \n",
       "        <td>Mutations</td>\n",
       "      \n",
       "        <td>Comprehensive genomic profiling of 28 metastatic ACCs.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acyc_fmi_2014_mutations</td>\n",
       "      \n",
       "        <td>acyc_fmi_2014</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>17</td>\n",
       "      \n",
       "        <td>MUTATION_EXTENDED</td>\n",
       "      \n",
       "        <td>MAF</td>\n",
       "      \n",
       "        <td>Mutations</td>\n",
       "      \n",
       "        <td>WGS of 28 primary salivary ACCs</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acyc_mda_2015_mutations</td>\n",
       "      \n",
       "        <td>acyc_mda_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>18</td>\n",
       "      \n",
       "        <td>COPY_NUMBER_ALTERATION</td>\n",
       "      \n",
       "        <td>DISCRETE</td>\n",
       "      \n",
       "        <td>Putative copy-number alterations from GISTIC</td>\n",
       "      \n",
       "        <td>Putative copy-number from GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acyc_mskcc_2013_cna</td>\n",
       "      \n",
       "        <td>acyc_mskcc_2013</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>19</td>\n",
       "      \n",
       "        <td>MUTATION_EXTENDED</td>\n",
       "      \n",
       "        <td>MAF</td>\n",
       "      \n",
       "        <td>Mutations</td>\n",
       "      \n",
       "        <td>Somatic mutations from exome sequencing of 60 tumor/normal pairs.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acyc_mskcc_2013_mutations</td>\n",
       "      \n",
       "        <td>acyc_mskcc_2013</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>20</td>\n",
       "      \n",
       "        <td>MUTATION_EXTENDED</td>\n",
       "      \n",
       "        <td>MAF</td>\n",
       "      \n",
       "        <td>Mutations</td>\n",
       "      \n",
       "        <td>Whole exome sequencing of 24 ACCs.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>acyc_sanger_2013_mutations</td>\n",
       "      \n",
       "        <td>acyc_sanger_2013</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>21</td>\n",
       "      \n",
       "        <td>COPY_NUMBER_ALTERATION</td>\n",
       "      \n",
       "        <td>DISCRETE</td>\n",
       "      \n",
       "        <td>Putative copy-number alterations from GISTIC</td>\n",
       "      \n",
       "        <td>Putative copy-number calls on 782 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub_cna</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>22</td>\n",
       "      \n",
       "        <td>MRNA_EXPRESSION</td>\n",
       "      \n",
       "        <td>CONTINUOUS</td>\n",
       "      \n",
       "        <td>mRNA expression (RNA-Seq RPKM)</td>\n",
       "      \n",
       "        <td>Expression levels for 26136 genes in 203 all cases (RNA-Seq RPKM).</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub_rna_seq_mrna</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>23</td>\n",
       "      \n",
       "        <td>MRNA_EXPRESSION</td>\n",
       "      \n",
       "        <td>Z-SCORE</td>\n",
       "      \n",
       "        <td>mRNA Expression z-Scores (RNA Seq RPKM)</td>\n",
       "      \n",
       "        <td>mRNA z-Scores (RNA Seq RPKM) compared to the expression distribution of each gene tumors that are diploid for this gene.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub_rna_seq_mrna_median_Zscores</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>24</td>\n",
       "      \n",
       "        <td>MRNA_EXPRESSION</td>\n",
       "      \n",
       "        <td>CONTINUOUS</td>\n",
       "      \n",
       "        <td>mRNA expression (microarray)</td>\n",
       "      \n",
       "        <td>mRNA expression levels (Affymetrix microarray)</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub_mrna</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>25</td>\n",
       "      \n",
       "        <td>MRNA_EXPRESSION</td>\n",
       "      \n",
       "        <td>Z-SCORE</td>\n",
       "      \n",
       "        <td>mRNA Expression z-Scores (microarray)</td>\n",
       "      \n",
       "        <td>mRNA z-Scores (Agilent microarray) compared to the expression distribution of each gene tumors that are diploid for this gene.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub_mrna_median_Zscores</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>26</td>\n",
       "      \n",
       "        <td>METHYLATION</td>\n",
       "      \n",
       "        <td>CONTINUOUS</td>\n",
       "      \n",
       "        <td>Methylation (hm27)</td>\n",
       "      \n",
       "        <td>Methylation beta-values (hm27 assay). For genes with multiple methylation probes, the probe least correlated with expression is selected.</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub_methylation_hm27</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>27</td>\n",
       "      \n",
       "        <td>MUTATION_EXTENDED</td>\n",
       "      \n",
       "        <td>MAF</td>\n",
       "      \n",
       "        <td>Somatic mutations</td>\n",
       "      \n",
       "        <td>Somatic mutation status from whole genome or whole exome sequencing on 298 samples.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub_mutations</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>28</td>\n",
       "      \n",
       "        <td>MRNA_EXPRESSION</td>\n",
       "      \n",
       "        <td>CONTINUOUS</td>\n",
       "      \n",
       "        <td>microRNA expression</td>\n",
       "      \n",
       "        <td>Expression levels for microRNAs.</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub_mirna</td>\n",
       "      \n",
       "        <td>all_phase2_target_2018_pub</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>29</td>\n",
       "      \n",
       "        <td>MUTATION_EXTENDED</td>\n",
       "      \n",
       "        <td>MAF</td>\n",
       "      \n",
       "        <td>Mutations</td>\n",
       "      \n",
       "        <td>Mutation status from exome sequencing in 44 samples.</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>all_stjude_2013_mutations</td>\n",
       "      \n",
       "        <td>all_stjude_2013</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "\n",
       "  \n",
       "    <tr>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "    </tr>\n",
       "\n",
       "    \n",
       "\n",
       "    <tr>\n",
       "      <td>885</td>\n",
       "        \n",
       "          <td>MRNA_EXPRESSION</td>\n",
       "        \n",
       "          <td>CONTINUOUS</td>\n",
       "        \n",
       "          <td>microRNA expression</td>\n",
       "        \n",
       "          <td>Expression levels for microRNAs.</td>\n",
       "        \n",
       "          <td>false</td>\n",
       "        \n",
       "          <td>wt_target_2018_pub_mirna</td>\n",
       "        \n",
       "          <td>wt_target_2018_pub</td>\n",
       "        \n",
       "    </tr>\n",
       "  \n",
       "</tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "#<Daru::DataFrame(886x7)>\n",
       "            molecularA   datatype       name descriptio showProfil molecularP    studyId\n",
       "          0 COPY_NUMBE   DISCRETE Putative c Putative c       true acbc_mskcc acbc_mskcc\n",
       "          1 MUTATION_E        MAF  Mutations Whole exom       true acbc_mskcc acbc_mskcc\n",
       "          2 PROTEIN_LE LOG2-VALUE Protein ex Protein ex      false acc_tcga_r   acc_tcga\n",
       "          3 PROTEIN_LE    Z-SCORE Protein ex Protein ex       true acc_tcga_r   acc_tcga\n",
       "          4 COPY_NUMBE   DISCRETE Putative c Putative c       true acc_tcga_g   acc_tcga\n",
       "          5 MRNA_EXPRE CONTINUOUS mRNA expre Expression      false acc_tcga_r   acc_tcga\n",
       "          6 MRNA_EXPRE    Z-SCORE mRNA Expre mRNA z-Sco       true acc_tcga_r   acc_tcga\n",
       "          7 COPY_NUMBE CONTINUOUS Relative l Relative l      false acc_tcga_l   acc_tcga\n",
       "          8 METHYLATIO CONTINUOUS Methylatio Methylatio      false acc_tcga_m   acc_tcga\n",
       "          9 MUTATION_E        MAF  Mutations Mutation d       true acc_tcga_m   acc_tcga\n",
       "         10 COPY_NUMBE   DISCRETE Putative c Putative c       true acc_tcga_p acc_tcga_p\n",
       "         11 MRNA_EXPRE CONTINUOUS mRNA Expre mRNA Expre      false acc_tcga_p acc_tcga_p\n",
       "         12 MRNA_EXPRE    Z-SCORE mRNA Expre mRNA Expre       true acc_tcga_p acc_tcga_p\n",
       "         13 COPY_NUMBE LOG2-VALUE Log2 copy- Log2 copy-      false acc_tcga_p acc_tcga_p\n",
       "         14 MUTATION_E        MAF  Mutations Mutation d       true acc_tcga_p acc_tcga_p\n",
       "        ...        ...        ...        ...        ...        ...        ...        ..."
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "C.get_all_molecular_profiles"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<b> Daru::DataFrame(9743x7) </b>\n",
       "<table>\n",
       "  <thead>\n",
       "  \n",
       "    <tr>\n",
       "      <th></th>\n",
       "      \n",
       "        <th>displayName</th>\n",
       "      \n",
       "        <th>description</th>\n",
       "      \n",
       "        <th>datatype</th>\n",
       "      \n",
       "        <th>patientAttribute</th>\n",
       "      \n",
       "        <th>priority</th>\n",
       "      \n",
       "        <th>clinicalAttributeId</th>\n",
       "      \n",
       "        <th>studyId</th>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "</thead>\n",
       "  <tbody>\n",
       "  \n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      \n",
       "        <td>Adjuvant Chemotherapy</td>\n",
       "      \n",
       "        <td>Adjuvant Chemotherapy</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>ADJUVANT_CHEMO</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      \n",
       "        <td>Adjuvant Treatment</td>\n",
       "      \n",
       "        <td>Adjuvant treatment.</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>ADJUVANT_TX</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      \n",
       "        <td>Diagnosis Age</td>\n",
       "      \n",
       "        <td>Age at which a condition or disease was first diagnosed.</td>\n",
       "      \n",
       "        <td>NUMBER</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>AGE</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      \n",
       "        <td>Cancer Type</td>\n",
       "      \n",
       "        <td>Disease type</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>CANCER_TYPE</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      \n",
       "        <td>Cancer Type Detailed</td>\n",
       "      \n",
       "        <td>Cancer Type Detailed</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>CANCER_TYPE_DETAILED</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>5</td>\n",
       "      \n",
       "        <td>ER Status By IHC</td>\n",
       "      \n",
       "        <td>breast carcinoma progesterone receptor status</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>ER_STATUS_BY_IHC</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>6</td>\n",
       "      \n",
       "        <td>Followup (years)</td>\n",
       "      \n",
       "        <td>Followup in years</td>\n",
       "      \n",
       "        <td>NUMBER</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>FOLLOWUP_YEARS</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>7</td>\n",
       "      \n",
       "        <td>Fusion Status</td>\n",
       "      \n",
       "        <td>Fusion Status</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>FUSION_STATUS</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>8</td>\n",
       "      \n",
       "        <td>Genomic Alterations</td>\n",
       "      \n",
       "        <td>Number of genomic alterations</td>\n",
       "      \n",
       "        <td>NUMBER</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>GENOMIC_ALTERATIONS</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>9</td>\n",
       "      \n",
       "        <td>IHC-HER2</td>\n",
       "      \n",
       "        <td>IHC-HER2</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>IHC_HER2</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>10</td>\n",
       "      \n",
       "        <td>Local Regional Recurrence</td>\n",
       "      \n",
       "        <td>Local Regional Recurrence</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>LOCAL_REGIONAL_RECURRENCE</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>11</td>\n",
       "      \n",
       "        <td>Metastatic Site</td>\n",
       "      \n",
       "        <td>The anatomic location to which the cancer has spread.</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>METASTATIC_SITE</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>12</td>\n",
       "      \n",
       "        <td>Metastatic tumor indicator</td>\n",
       "      \n",
       "        <td>Distant metastasis present ind2</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>METASTATIC_TUMOR_INDICATOR</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>13</td>\n",
       "      \n",
       "        <td>Mutation Count</td>\n",
       "      \n",
       "        <td>Mutation Count</td>\n",
       "      \n",
       "        <td>NUMBER</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>30</td>\n",
       "      \n",
       "        <td>MUTATION_COUNT</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>14</td>\n",
       "      \n",
       "        <td>MYB or MYBL1 Fusion</td>\n",
       "      \n",
       "        <td>Description of MYB or MYBL1 fusions identified.</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>MYB_MYBL1_FUSION</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>15</td>\n",
       "      \n",
       "        <td>MYB-NFIB Fusion Gene Copy Number Alterations</td>\n",
       "      \n",
       "        <td>MYB-NFIB Fusion Gene Copy Number Alterations</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>MYB_NFIB_CNA</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>16</td>\n",
       "      \n",
       "        <td>MYB-NFIB FISH</td>\n",
       "      \n",
       "        <td>MYB-NFIB FISH.</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>MYB_NFIB_FISH</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>17</td>\n",
       "      \n",
       "        <td>MYB-NFIB Fusion Gene Non-synonymous Mutations</td>\n",
       "      \n",
       "        <td>Count of non-synonymous mutations in MYB-NFIB Fusion Gene</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>MYB_NFIB_NONSYNONYMOUS_COUNT</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>18</td>\n",
       "      \n",
       "        <td>MYB-NFIB Rearrangement</td>\n",
       "      \n",
       "        <td>MYB-NFIB Rearrangement Description</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>MYB_NFIB_REARRANGEMENT</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>19</td>\n",
       "      \n",
       "        <td>Neoadjuvant Chemotherapy</td>\n",
       "      \n",
       "        <td>Neoadjuvant Chemotherapy.</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>NEOADJUVANT_CHEMO</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>20</td>\n",
       "      \n",
       "        <td>Oncotree Code</td>\n",
       "      \n",
       "        <td>Oncotree Code</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>ONCOTREE_CODE</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>21</td>\n",
       "      \n",
       "        <td>Overall Patient Histology</td>\n",
       "      \n",
       "        <td>Overall Patient Histology</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>OVERALL_PATIENT_HISTOLOGY</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>22</td>\n",
       "      \n",
       "        <td>Platform</td>\n",
       "      \n",
       "        <td>Platform.</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>PLATFORM</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>23</td>\n",
       "      \n",
       "        <td>Primary Tumor Site</td>\n",
       "      \n",
       "        <td>Text term to describe the organ sub-division in an individual with cancer.</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>PRIMARY_SITE</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>24</td>\n",
       "      \n",
       "        <td>PR status by ihc</td>\n",
       "      \n",
       "        <td>Breast carcinoma progesterone receptor status</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>PR_STATUS_BY_IHC</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>25</td>\n",
       "      \n",
       "        <td>Radiation Therapy</td>\n",
       "      \n",
       "        <td>Radiation therapy description</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>RADIATION_THERAPY</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>26</td>\n",
       "      \n",
       "        <td>Number of Samples Per Patient</td>\n",
       "      \n",
       "        <td>Number of Samples Per Patient</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>SAMPLE_COUNT</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>27</td>\n",
       "      \n",
       "        <td>Sample Type</td>\n",
       "      \n",
       "        <td>\"The type of sample (i.e., normal, primary, met, recurrence).\"</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>false</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>SAMPLE_TYPE</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>28</td>\n",
       "      \n",
       "        <td>Sex</td>\n",
       "      \n",
       "        <td>Sex</td>\n",
       "      \n",
       "        <td>STRING</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>SEX</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>29</td>\n",
       "      \n",
       "        <td>Time to Metastasis (months)</td>\n",
       "      \n",
       "        <td>Time to metastasis (months).</td>\n",
       "      \n",
       "        <td>NUMBER</td>\n",
       "      \n",
       "        <td>true</td>\n",
       "      \n",
       "        <td>1</td>\n",
       "      \n",
       "        <td>TIME_TO_MET_MONTHS</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "\n",
       "  \n",
       "    <tr>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "    </tr>\n",
       "\n",
       "    \n",
       "\n",
       "    <tr>\n",
       "      <td>9742</td>\n",
       "        \n",
       "          <td>Sex</td>\n",
       "        \n",
       "          <td>Sex</td>\n",
       "        \n",
       "          <td>STRING</td>\n",
       "        \n",
       "          <td>true</td>\n",
       "        \n",
       "          <td>1</td>\n",
       "        \n",
       "          <td>SEX</td>\n",
       "        \n",
       "          <td>wt_target_2018_pub</td>\n",
       "        \n",
       "    </tr>\n",
       "  \n",
       "</tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "#<Daru::DataFrame(9743x7)>\n",
       "            displayNam descriptio   datatype patientAtt   priority clinicalAt    studyId\n",
       "          0 Adjuvant C Adjuvant C     STRING       true          1 ADJUVANT_C acbc_mskcc\n",
       "          1 Adjuvant T Adjuvant t     STRING       true          1 ADJUVANT_T acbc_mskcc\n",
       "          2 Diagnosis  Age at whi     NUMBER       true          1        AGE acbc_mskcc\n",
       "          3 Cancer Typ Disease ty     STRING      false          1 CANCER_TYP acbc_mskcc\n",
       "          4 Cancer Typ Cancer Typ     STRING      false          1 CANCER_TYP acbc_mskcc\n",
       "          5 ER Status  breast car     STRING      false          1 ER_STATUS_ acbc_mskcc\n",
       "          6 Followup ( Followup i     NUMBER       true          1 FOLLOWUP_Y acbc_mskcc\n",
       "          7 Fusion Sta Fusion Sta     STRING      false          1 FUSION_STA acbc_mskcc\n",
       "          8 Genomic Al Number of      NUMBER      false          1 GENOMIC_AL acbc_mskcc\n",
       "          9   IHC-HER2   IHC-HER2     STRING      false          1   IHC_HER2 acbc_mskcc\n",
       "         10 Local Regi Local Regi     STRING       true          1 LOCAL_REGI acbc_mskcc\n",
       "         11 Metastatic The anatom     STRING      false          1 METASTATIC acbc_mskcc\n",
       "         12 Metastatic Distant me     STRING      false          1 METASTATIC acbc_mskcc\n",
       "         13 Mutation C Mutation C     NUMBER      false         30 MUTATION_C acbc_mskcc\n",
       "         14 MYB or MYB Descriptio     STRING       true          1 MYB_MYBL1_ acbc_mskcc\n",
       "        ...        ...        ...        ...        ...        ...        ...        ..."
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "C.get_all_clinical_attributes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<b> Daru::DataFrame(17x2) </b>\n",
       "<table>\n",
       "  <thead>\n",
       "  \n",
       "    <tr>\n",
       "      <th></th>\n",
       "      \n",
       "        <th>description</th>\n",
       "      \n",
       "        <th>genePanelId</th>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "</thead>\n",
       "  <tbody>\n",
       "  \n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      \n",
       "        <td>Targeted NGS of NSCLC Samples.</td>\n",
       "      \n",
       "        <td>NSCLC_UNITO_2016_PANEL</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      \n",
       "        <td>Targeted (341 cancer genes) sequencing of various tumor types via MSK-IMPACT on Illumina HiSeq sequencers.</td>\n",
       "      \n",
       "        <td>IMPACT341</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      \n",
       "        <td>Targeted (410 cancer genes) sequencing of various tumor types via MSK-IMPACT on Illumina HiSeq sequencers.</td>\n",
       "      \n",
       "        <td>IMPACT410</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      \n",
       "        <td>Targeted sequencing of various tumor types via bait v4.</td>\n",
       "      \n",
       "        <td>bait_v4</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      \n",
       "        <td>Targeted sequencing of various tumor types via bait v5.</td>\n",
       "      \n",
       "        <td>bait_v5</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>5</td>\n",
       "      \n",
       "        <td>Targeted sequencing of various tumor types via bait v3.</td>\n",
       "      \n",
       "        <td>bait_v3</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>6</td>\n",
       "      \n",
       "        <td>Targeted (468 cancer genes) sequencing of various tumor types via MSK-IMPACT on Illumina HiSeq sequencers.</td>\n",
       "      \n",
       "        <td>IMPACT468</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>7</td>\n",
       "      \n",
       "        <td>Targeted sequencing of urcc tumor via MSK-IMPACT.</td>\n",
       "      \n",
       "        <td>IMPACT</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>8</td>\n",
       "      \n",
       "        <td>Targeted sequencing of 15 oncogenes and tumor suppressor genes, selected for their potential as predictive biomarkers of response to targeted agents by Sanger sequencing.</td>\n",
       "      \n",
       "        <td>MSKCC_BLCA_15</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>9</td>\n",
       "      \n",
       "        <td>Targeted (300 cancer genes) sequencing of bladder urothelial carcinomas via MSK-IMPACT on Illumina HiSeq sequencers.</td>\n",
       "      \n",
       "        <td>IMPACT300</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>10</td>\n",
       "      \n",
       "        <td>Targeted sequencing of 504 cancer-associated genes on Illumina HiSeq sequencers.</td>\n",
       "      \n",
       "        <td>DFCI_504</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>11</td>\n",
       "      \n",
       "        <td>Targeted deep sequencing (TDS) of 1202 genes on Illumina HiSeq2000 sequencers.</td>\n",
       "      \n",
       "        <td>UCLA_1202</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>12</td>\n",
       "      \n",
       "        <td>Targeted (279 cancer genes) sequencing of various tumor types via MSK-IMPACT on Illumina HiSeq sequencers.</td>\n",
       "      \n",
       "        <td>IMPACT279</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>13</td>\n",
       "      \n",
       "        <td>DNA sequencing of 623 genes with known or potential relationships to cancer.</td>\n",
       "      \n",
       "        <td>WUSTL-DFCI_623</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>14</td>\n",
       "      \n",
       "        <td>Targeted massively parallel sequencing of 1651 cancer genes.</td>\n",
       "      \n",
       "        <td>NOVARTIS_BROAD_1651</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>15</td>\n",
       "      \n",
       "        <td>Targeted (27 cancer genes) sequencing of adenoid cystic carcinomas on Illumina HiSeq sequencers.</td>\n",
       "      \n",
       "        <td>ACYC_FMI_27</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>16</td>\n",
       "      \n",
       "        <td>Targeted (173 cancer genes) sequencing of breast cancers on Illumina HiSeq sequencers.</td>\n",
       "      \n",
       "        <td>METABRIC_173</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "\n",
       "  \n",
       "</tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "#<Daru::DataFrame(17x2)>\n",
       "            descriptio genePanelI\n",
       "          0 Targeted N NSCLC_UNIT\n",
       "          1 Targeted (  IMPACT341\n",
       "          2 Targeted (  IMPACT410\n",
       "          3 Targeted s    bait_v4\n",
       "          4 Targeted s    bait_v5\n",
       "          5 Targeted s    bait_v3\n",
       "          6 Targeted (  IMPACT468\n",
       "          7 Targeted s     IMPACT\n",
       "          8 Targeted s MSKCC_BLCA\n",
       "          9 Targeted (  IMPACT300\n",
       "         10 Targeted s   DFCI_504\n",
       "         11 Targeted d  UCLA_1202\n",
       "         12 Targeted (  IMPACT279\n",
       "         13 DNA sequen WUSTL-DFCI\n",
       "         14 Targeted m NOVARTIS_B\n",
       "        ...        ...        ..."
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "C.get_all_gene_panels"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<b> Daru::DataFrame(1359x5) </b>\n",
       "<table>\n",
       "  <thead>\n",
       "  \n",
       "    <tr>\n",
       "      <th></th>\n",
       "      \n",
       "        <th>category</th>\n",
       "      \n",
       "        <th>name</th>\n",
       "      \n",
       "        <th>description</th>\n",
       "      \n",
       "        <th>sampleListId</th>\n",
       "      \n",
       "        <th>studyId</th>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "</thead>\n",
       "  <tbody>\n",
       "  \n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      \n",
       "        <td>all_cases_with_mutation_data</td>\n",
       "      \n",
       "        <td>Sequenced Tumors</td>\n",
       "      \n",
       "        <td>All (Next-Gen) sequenced samples (12 samples)</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015_sequenced</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      \n",
       "        <td>all_cases_with_mutation_and_cna_data</td>\n",
       "      \n",
       "        <td>Tumor Samples with sequencing and CNA data</td>\n",
       "      \n",
       "        <td>All tumor samples that have CNA and sequencing data (12 samples)</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015_cnaseq</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      \n",
       "        <td>all_cases_with_cna_data</td>\n",
       "      \n",
       "        <td>Tumor Samples with CNA data</td>\n",
       "      \n",
       "        <td>All tumors with CNA data (12 samples)</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015_cna</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      \n",
       "        <td>all_cases_in_study</td>\n",
       "      \n",
       "        <td>All Tumors</td>\n",
       "      \n",
       "        <td>All tumor samples (12 samples)</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015_all</td>\n",
       "      \n",
       "        <td>acbc_mskcc_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      \n",
       "        <td>all_cases_with_methylation_data</td>\n",
       "      \n",
       "        <td>Tumor Samples with methylation data (HM450)</td>\n",
       "      \n",
       "        <td>All samples with methylation (HM450) data (80 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_methylation_hm450</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>5</td>\n",
       "      \n",
       "        <td>all_cases_with_methylation_data</td>\n",
       "      \n",
       "        <td>All tumor samples with methylation data</td>\n",
       "      \n",
       "        <td>All samples with methylation data (80 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_methylation_all</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>6</td>\n",
       "      \n",
       "        <td>all_cases_with_mutation_data</td>\n",
       "      \n",
       "        <td>All Sequenced Tumors</td>\n",
       "      \n",
       "        <td>All sequenced tumor samples (90 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_sequenced</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>7</td>\n",
       "      \n",
       "        <td>all_cases_with_mutation_and_cna_data</td>\n",
       "      \n",
       "        <td>Tumor Samples with sequencing and CNA data</td>\n",
       "      \n",
       "        <td>All tumor samples that have CNA and sequencing data (88 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_cnaseq</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>8</td>\n",
       "      \n",
       "        <td>all_cases_with_cna_data</td>\n",
       "      \n",
       "        <td>Tumor Samples with CNA data</td>\n",
       "      \n",
       "        <td>All tumors with CNA data (90 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_cna</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>9</td>\n",
       "      \n",
       "        <td>all_cases_with_mrna_rnaseq_data</td>\n",
       "      \n",
       "        <td>Tumor Samples with mRNA data (RNA Seq V2)</td>\n",
       "      \n",
       "        <td>All samples with mRNA expression data (79 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_rna_seq_v2_mrna</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>10</td>\n",
       "      \n",
       "        <td>all_cases_in_study</td>\n",
       "      \n",
       "        <td>All Tumors</td>\n",
       "      \n",
       "        <td>All tumor samples (92 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_all</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>11</td>\n",
       "      \n",
       "        <td>all_cases_with_mutation_and_cna_and_mrna_data</td>\n",
       "      \n",
       "        <td>All Complete Tumors</td>\n",
       "      \n",
       "        <td>All tumor samples that have mRNA, CNA and sequencing data (75 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_3way_complete</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>12</td>\n",
       "      \n",
       "        <td>all_cases_with_rppa_data</td>\n",
       "      \n",
       "        <td>Tumor Samples with RPPA data</td>\n",
       "      \n",
       "        <td>Tumors with reverse phase protein array (RPPA) data for about 200 antibodies (46 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_rppa</td>\n",
       "      \n",
       "        <td>acc_tcga</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>13</td>\n",
       "      \n",
       "        <td>all_cases_with_mutation_data</td>\n",
       "      \n",
       "        <td>Sequenced Tumors</td>\n",
       "      \n",
       "        <td>All (Next-Gen) sequenced samples (91 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018_sequenced</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>14</td>\n",
       "      \n",
       "        <td>all_cases_with_mutation_and_cna_data</td>\n",
       "      \n",
       "        <td>Tumor Samples with sequencing and CNA data</td>\n",
       "      \n",
       "        <td>All tumor samples that have CNA and sequencing data (89 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018_cnaseq</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>15</td>\n",
       "      \n",
       "        <td>all_cases_with_cna_data</td>\n",
       "      \n",
       "        <td>Tumor Samples with CNA data</td>\n",
       "      \n",
       "        <td>All tumors with CNA data (89 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018_cna</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>16</td>\n",
       "      \n",
       "        <td>all_cases_with_mrna_rnaseq_data</td>\n",
       "      \n",
       "        <td>Tumor Samples with mRNA data (RNA Seq V2)</td>\n",
       "      \n",
       "        <td>All samples with mRNA expression data (78 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018_rna_seq_v2_mrna</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>17</td>\n",
       "      \n",
       "        <td>all_cases_in_study</td>\n",
       "      \n",
       "        <td>All Tumors</td>\n",
       "      \n",
       "        <td>All tumor samples (92 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018_all</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>18</td>\n",
       "      \n",
       "        <td>all_cases_with_log2_cna_data</td>\n",
       "      \n",
       "        <td>Tumor Samples with log2 copy-number</td>\n",
       "      \n",
       "        <td>All tumors with log2 copy-number data (89 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018_log2CNA</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>19</td>\n",
       "      \n",
       "        <td>all_cases_with_mutation_and_cna_and_mrna_data</td>\n",
       "      \n",
       "        <td>All Complete Tumors</td>\n",
       "      \n",
       "        <td>All tumor samples that have mRNA, CNA and sequencing data (76 samples)</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018_3way_complete</td>\n",
       "      \n",
       "        <td>acc_tcga_pan_can_atlas_2018</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>20</td>\n",
       "      \n",
       "        <td>all_cases_with_mutation_data</td>\n",
       "      \n",
       "        <td>Sequenced Tumors</td>\n",
       "      \n",
       "        <td>All (Next-Gen) sequenced samples (28 samples)</td>\n",
       "      \n",
       "        <td>acyc_fmi_2014_sequenced</td>\n",
       "      \n",
       "        <td>acyc_fmi_2014</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>21</td>\n",
       "      \n",
       "        <td>all_cases_with_mutation_and_cna_data</td>\n",
       "      \n",
       "        <td>Tumor Samples with sequencing and CNA data</td>\n",
       "      \n",
       "        <td>All tumor samples that have CNA and sequencing data (28 samples)</td>\n",
       "      \n",
       "        <td>acyc_fmi_2014_cnaseq</td>\n",
       "      \n",
       "        <td>acyc_fmi_2014</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>22</td>\n",
       "      \n",
       "        <td>all_cases_with_cna_data</td>\n",
       "      \n",
       "        <td>Tumor Samples with CNA data</td>\n",
       "      \n",
       "        <td>All tumors with CNA data (28 samples)</td>\n",
       "      \n",
       "        <td>acyc_fmi_2014_cna</td>\n",
       "      \n",
       "        <td>acyc_fmi_2014</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>23</td>\n",
       "      \n",
       "        <td>all_cases_in_study</td>\n",
       "      \n",
       "        <td>All Tumors</td>\n",
       "      \n",
       "        <td>All tumor samples (28 samples)</td>\n",
       "      \n",
       "        <td>acyc_fmi_2014_all</td>\n",
       "      \n",
       "        <td>acyc_fmi_2014</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>24</td>\n",
       "      \n",
       "        <td>all_cases_with_mutation_data</td>\n",
       "      \n",
       "        <td>Sequenced Tumors</td>\n",
       "      \n",
       "        <td>All (Next-Gen) sequenced samples (65 samples)</td>\n",
       "      \n",
       "        <td>acyc_mda_2015_sequenced</td>\n",
       "      \n",
       "        <td>acyc_mda_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>25</td>\n",
       "      \n",
       "        <td>all_cases_in_study</td>\n",
       "      \n",
       "        <td>All Tumors</td>\n",
       "      \n",
       "        <td>All tumor samples (102 samples)</td>\n",
       "      \n",
       "        <td>acyc_mda_2015_all</td>\n",
       "      \n",
       "        <td>acyc_mda_2015</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>26</td>\n",
       "      \n",
       "        <td>all_cases_with_mutation_data</td>\n",
       "      \n",
       "        <td>Sequenced Tumors</td>\n",
       "      \n",
       "        <td>All sequenced samples (60 samples)</td>\n",
       "      \n",
       "        <td>acyc_mskcc_2013_sequenced</td>\n",
       "      \n",
       "        <td>acyc_mskcc_2013</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>27</td>\n",
       "      \n",
       "        <td>all_cases_with_mutation_and_cna_data</td>\n",
       "      \n",
       "        <td>Tumors with sequencing and CNA data</td>\n",
       "      \n",
       "        <td>All tumor samples that have CNA and sequencing data (60 samples)</td>\n",
       "      \n",
       "        <td>acyc_mskcc_2013_cnaseq</td>\n",
       "      \n",
       "        <td>acyc_mskcc_2013</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>28</td>\n",
       "      \n",
       "        <td>all_cases_with_cna_data</td>\n",
       "      \n",
       "        <td>Tumors CNA</td>\n",
       "      \n",
       "        <td>All tumors with CNA data (60 samples)</td>\n",
       "      \n",
       "        <td>acyc_mskcc_2013_cna</td>\n",
       "      \n",
       "        <td>acyc_mskcc_2013</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "    <tr>\n",
       "      <td>29</td>\n",
       "      \n",
       "        <td>all_cases_in_study</td>\n",
       "      \n",
       "        <td>All Tumors</td>\n",
       "      \n",
       "        <td>All tumor samples (60 samples)</td>\n",
       "      \n",
       "        <td>acyc_mskcc_2013_all</td>\n",
       "      \n",
       "        <td>acyc_mskcc_2013</td>\n",
       "      \n",
       "    </tr>\n",
       "  \n",
       "\n",
       "  \n",
       "    <tr>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "        <td>...</td>\n",
       "      \n",
       "    </tr>\n",
       "\n",
       "    \n",
       "\n",
       "    <tr>\n",
       "      <td>1358</td>\n",
       "        \n",
       "          <td>all_cases_with_mutation_and_cna_and_mrna_data</td>\n",
       "        \n",
       "          <td>All Complete Tumors</td>\n",
       "        \n",
       "          <td>All tumor samples that have mRNA, CNA and sequencing data (101 samples)</td>\n",
       "        \n",
       "          <td>wt_target_2018_pub_3way_complete</td>\n",
       "        \n",
       "          <td>wt_target_2018_pub</td>\n",
       "        \n",
       "    </tr>\n",
       "  \n",
       "</tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "#<Daru::DataFrame(1359x5)>\n",
       "              category       name descriptio sampleList    studyId\n",
       "          0 all_cases_ Sequenced  All (Next- acbc_mskcc acbc_mskcc\n",
       "          1 all_cases_ Tumor Samp All tumor  acbc_mskcc acbc_mskcc\n",
       "          2 all_cases_ Tumor Samp All tumors acbc_mskcc acbc_mskcc\n",
       "          3 all_cases_ All Tumors All tumor  acbc_mskcc acbc_mskcc\n",
       "          4 all_cases_ Tumor Samp All sample acc_tcga_m   acc_tcga\n",
       "          5 all_cases_ All tumor  All sample acc_tcga_m   acc_tcga\n",
       "          6 all_cases_ All Sequen All sequen acc_tcga_s   acc_tcga\n",
       "          7 all_cases_ Tumor Samp All tumor  acc_tcga_c   acc_tcga\n",
       "          8 all_cases_ Tumor Samp All tumors acc_tcga_c   acc_tcga\n",
       "          9 all_cases_ Tumor Samp All sample acc_tcga_r   acc_tcga\n",
       "         10 all_cases_ All Tumors All tumor  acc_tcga_a   acc_tcga\n",
       "         11 all_cases_ All Comple All tumor  acc_tcga_3   acc_tcga\n",
       "         12 all_cases_ Tumor Samp Tumors wit acc_tcga_r   acc_tcga\n",
       "         13 all_cases_ Sequenced  All (Next- acc_tcga_p acc_tcga_p\n",
       "         14 all_cases_ Tumor Samp All tumor  acc_tcga_p acc_tcga_p\n",
       "        ...        ...        ...        ...        ...        ..."
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "C.get_all_sample_lists"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "@webio": {
   "lastCommId": "c0ade92883c94f0d8683c1d6ed69ad10",
   "lastKernelId": "eab37806-1e8b-4d96-b9f8-11f09a64be03"
  },
  "kernelspec": {
   "display_name": "Ruby 2.6.0",
   "language": "ruby",
   "name": "ruby"
  },
  "language_info": {
   "file_extension": ".rb",
   "mimetype": "application/x-ruby",
   "name": "ruby",
   "version": "2.6.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
